### PR TITLE
Ft mpesa endpoint

### LINKED ISSUE #8: Mpesa Daraja Payment Gateway
**Tasks**
- [x] Create an account on developers.safaricom.co.ke.
- [x] Integrate Mpesa Daraja.
- [ ] Wire up webhooks for callbacks.
- [ ] Write tests ready backend endpoints
- [ ] Host the backend app on Heroku

### PR DESCRIPTION
**Pull request by moseti1**
This feature will enable application to access various mpesa transaction services.Currently able to request and return for mpesa access token. Improvement still in progress.

### CODE REVIEW

File: app/api/v1/utils/mpesa_transact.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+app.config["API_ENVIRONMENT"] = "sandbox" #sandbox or live
+app.config["APP_KEY"] = consumer_key # App_key from developers portal
+app.config["APP_SECRET"] = consumer_key #App_Secret from developers portal
+
+
+
+class MpesaTransact():
+
```
Comment (Dave-mash) [inline app/api/v1/utils/mpesa_transact.py]: Neat work 👏

File: templates/home.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (Dave-mash) [inline templates/home.html]: Why do you need this file?

Comment (moseti1) [reply templates/home.html]: The file is useless now. I used it to see content display in browser.Will be dropped in the next  changes.

**Review by Dave-mash — APPROVED** [review]
Keep up the good work

**Review by Jordan-type — APPROVED** [review]
Awesome work 👍🏽

**Pull request merged**